### DISCUSSION TITLE
Bots confuse medical treatment with being attacked.

### DISCUSSION THREAD

**Discussion by NecroCloakDozer** [FakeFishGames/Barotrauma#14158] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After a Thresher attack, I give the crewmate two doses of antibiotics to get rid of their infected wounds. Immediately after, security enrages and knocks me out, handcuffing me. 

I did a search for a similar discussion, but that one points to foreign NPCs becoming enraged, this one I have only experienced with my own crew. 

### Reproduction steps

1. Spawn a security bot.
2. Give the bot infected wound.
3. Treat with antibiotics.
4. Have bots confuse antibiotics with poison and attack you.
5. Get handcuffed. 

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.7.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14101
